onnxruntime/core/providers/webnn/builders/helper.cc

```diff
@@ -99,6 +99,70 @@ bool IsTensorShapeSupported(const NodeArg& node_arg, const std::string& parent_n
   return true;
 }
 
+// Check if all input tensor ranks of the given node are supported by WebNN.
+bool IsInputRankSupportedByOp(const Node& node, const emscripten::val& wnn_limits, const logging::Logger& logger) {
+  const std::string_view op_type = node.OpType();
+
+  auto it = op_inputs_map.find(op_type);
+  if (it == op_inputs_map.end()) {
+    LOGS(logger, VERBOSE) << "Operator type: [" << op_type << "] is not found in the op inputs map.";
+    return false;
+  }
+
+  for (const auto& input : it->second.inputs) {
+    const std::string_view webnn_op_type = it->second.opType;
+
+    if (static_cast<size_t>(input.index) >= node.InputDefs().size() || node.InputDefs()[input.index] == nullptr) {
+      LOGS(logger, VERBOSE) << "Operator type: [" << op_type
+                            << "], input index: [" << input.index
+                            << "], corresponding WebNN op type: " << webnn_op_type
+                            << ", WebNN input name " << input.name
+                            << "] is not found.";
```

Comment:
Error message copy pasta. It should say than the input index is invalid (or unsupported) rather than the operator is not found.